### DISCUSSION TITLE
Prevent kernel function calling

### DISCUSSION THREAD

**Discussion by mickaelropars** [microsoft/semantic-kernel#9308] [General]
I have some data pushed into prompt that contains {{Variable}} . this variable is not function 

but when creating the template semantic kernel is trying to call the kernel function "Variable"

is there a method in order to prevent these behavior or should I sanitize the prompt before rendering it ?

Micka

**Comment by sophialagerkranspandey** [comment]
The syntax should be {{$Variable}}

**Reply by mickaelropars** [reply]
@sophialagerkranspandey Thanks a lot Sophia for your answer , My goal is to prevent the call of kernel function in this case, because the data I am passing within the context contains data that are not kernel variables or function but have the same schema than the semantic kernel.
